SymPEP-template.md

```diff
@@ -0,0 +1,135 @@
+
+This section lists the major steps required to implement the SymPEP.  Where
+possible, it should be noted where one step is dependent on another, and which
+steps may be optionally omitted.  Where it makes sense, each step should
+include a link to related pull requests as the implementation progresses.
+
+Any pull requests or development branches containing work on this SymPEP should
+be linked to from here.  (A SymPEP does not need to be implemented in a single
+pull request if it makes sense to implement it in discrete phases).
+
+## Alternatives
+
+If there were any alternative solutions to solving the same problem, they should
+be discussed here, along with a justification for the chosen approach.
+
+## Discussion
+
+This section may just be a bullet list including links to any discussions
+regarding the SymPEP. All issues, pull requests, discussions, and mailing list
+threads relating to a SymPEP should be listed and hyperlinked here.
+
+## References
+
+Any references should be listed here. Note this does not include links to
+discussions about the SymPEP, which are listed in the previous section. For
+online resources, freely accessible and stable online resources such as
+Wikipedia, Wolfram MathWorld, and the NIST Digital Library of Mathematical
+Functions (DLMF), which are unlikely to suffer from hyperlink rot, should be
+preferred. Non-online references such as textbook or literature references may
+also be used here.
+
+## Copyright
+
+Each SymPEP must be explicitly labeled as placed in the public domain, using
+the below sentence (the below sentence also applies to this template).
+
+This document has been placed in the public domain.
+
+## Extra Notes About SymPEPs
+
+NOTE: This section is not part of the template. It should not be included in
+any SymPEPs. It is a list of notes of things that apply to all parts of the
+above template.
+
+Some extra notes about writing a SymPEP:
+
+- All SymPEPs live in the [SymPEPs
+  repository](https://github.com/sympy/SymPEPs/) on GitHub. All proposed
+  SymPEPs should be made as pull requests to that repo (see [SymPEP
+  1](SymPEP-0001)).
+- The number corresponding to a SymPEP will be assigned when it is first
+  proposed to the community. You may use `XXXX` as a placeholder number until
+  this is done.
+- The file for a SymPEPs should be named `SymPEP-XXXX.md` where `XXXX` is a
+  four digit number, preceded with leading 0s. It should be placed at the root
+  of the [SymPEPs repository](https://github.com/sympy/SymPEPs/).
+- All SymPEP documents should be written in Markdown, following this template.
+  The Markdown should be compatible with GitHub Flavored Markdown, so that the
+  SymPEPs are viewable on GitHub. This means that Markdown features
+  not supported by GitHub, such as footnotes, should be avoided.
+  <!-- XXX: Perhaps we should abandon this and only require them to be
+  readable in some rendered format. That would allow us to use MathJAX, which
+  could be useful. -->
```

Comment:
Math is supported in GitHub markdown now, so this can be removed. In fact, it might be worth pointing that out somewhere.